src/TimeSteppers/ssp_rk3.jl

```diff
@@ -0,0 +1,222 @@
+using KernelAbstractions: @kernel, @index
+
+using Oceananigans: AbstractModel, prognostic_fields
+using Oceananigans.Utils: launch!, time_difference_seconds
+
+using Oceananigans.TimeSteppers:
+    AbstractTimeStepper,
+    tick!,
+    update_state!,
+    compute_tendencies!,
+    compute_flux_bc_tendencies!,
+    compute_pressure_correction!,
+    make_pressure_correction!,
+    step_lagrangian_particles!
+
+import Oceananigans.TimeSteppers: time_step!
+
+"""
+    SSPRungeKutta3TimeStepper{FT, U0, TG, TI} <: AbstractTimeStepper
```

Comment:
```suggestion
$(TYPEDEF)
```

Reply:
what's your opinion about docstrings on structs? In some previous lifetime we decided they were not a good idea, since users have no use for them, and they pollute the help query.

Reply:
I think they are still useful, at least for other contributors/developers if these structs aren't meant to be consumed by end-users.

I understand the argument of polluting the search results, but perhaps that points to a higher infrastructure missing feature in Documenter, but I don't think the result should be to document _fewer_ things.

Reply:
i'll merge this but i want to see the docs build first